cdap-ui/app/cdap/components/Wizard/index.js

```diff
@@ -157,6 +149,39 @@ export default class Wizard extends Component{
       let state = this.props.store.getState()[stepId];
       return state && state.__complete;
     };
+    const onSubmitSuccessMessage = () => {
+      return (
```

Comment:
we should avoid assigning function to const 
ref: https://github.com/airbnb/javascript#functions